addon/pods/components/rui-error-template/template.hbs

```diff
@@ -14,5 +14,10 @@
     <div class="rui-error-template-tagline">
       {{tagline}}
     </div>
+    {{#if link}}
+      <a class='btn btn-secondary btn-lg rui-error-button' href={{link}}>
```

Comment:
okay, just thought about this a lot.... I was going to add some to the rev-ui that would check to see if a full URL existed, true would just use an anchor tag, or false (indicating a relative link) and the template would render the {{#link-to helper. BUT, since the link-to helper really just generates the relative links, we can achieve this by just passing through the relative URL to the way it is currently setup.
<img width="614" alt="screen shot 2016-07-18 at 3 27 59 pm" src="https://cloud.githubusercontent.com/assets/5505212/16932667/b618d34a-4cfc-11e6-8f66-da6675bc4c02.png">
...
For example, I can call the error template component like so:
```
{{rui-error-template
  status='401'
  tagline='You are unauthorized to be here.'
  link='/cli/ea/error'
  link-message='Login'}}
```
& then the relative link will appear.
I think it's a lot cleaner this way, and I don't really see a point in adding extra logic into the component, as well as dirtying up the template to achieve the same functionality.